libraries/botframework-schema/src/index.ts

```diff
@@ -2256,6 +2256,7 @@ export enum Channels {
     Kik = 'kik',
```

Comment:
in the comment  * Defines values for ChannelIds for Channels. it is missing the new value

Reply:
Nice catch! Thanks!